kudago-java/src/test/java/tbank/kudago/controller/unit/LocationControllerTest.java

```diff
@@ -97,11 +99,31 @@ public void testUpdateLocationById() throws Exception {
     }
 
     @Test
-    public void testDeleteLocationById() throws Exception {
+    public void successDeleteLocationById() throws Exception {
         Long id = 1L;
         doNothing().when(locationRepository).deleteById(id);
         mockMvc.perform(delete(API_URL + "/{id}", id))
                 .andExpect(status().isOk());
         verify(locationRepository, times(1)).deleteById(id);
     }
+
+    @Test
+    public void failureGetLocationById() throws Exception {
+        when(locationRepository.getById(1L)).thenReturn(Optional.empty());
+
+        mockMvc.perform(get(API_URL + "/1"))
+                .andExpect(status().isNotFound());
+    }
+
+    @Test
+    public void failurePutLocationById() throws Exception {
+        when(locationRepository.getById(1L)).thenReturn(Optional.empty());
+
+        Location location = new Location("slug", "name");
+
+        mockMvc.perform(put(API_URL + "/1")
```

Comment:
там кстати можно делать так  
```suggestion
        mockMvc.perform(put(API_URL + "/{id}", 1)
```

Reply:
да, так действительно лучше, исправила)